tools/breeze/src/main.rs

```diff
@@ -5,7 +5,7 @@ use cgos::congatec::Congatec;
 use cgos::status::Status;
 
 const FAN_MAX_SPEED: f32 = 100.0;
-const FAN_MIN_SPEED: f32 = 30.0;
+const FAN_MIN_SPEED: f32 = 40.0;
```

Comment:
Should we add a note why 40 is a threshold here?

Reply:
Commentified :wink: